auth-api/poetry.lock

```diff
@@ -1,114 +1,99 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.4 and should not be changed by hand.
```

Comment:
Updating all the deps, probably should require regression before pushing to PROD..

Is it possible to just update the deps needed?